RadixWallet/Clients/AppPreferencesClient/AppPreferencesClient+Interface.swift

```diff
@@ -70,6 +70,14 @@ extension AppPreferencesClient {
 			display.isCurrencyAmountVisible.toggle()
 		}
 	}
+
+	public func getHiddenAssets() async -> [AssetAddress] {
+		await getPreferences().assets.hiddenAssets
+	}
+
+	public func isAssetHidden(asset: AssetAddress) async -> Bool {
```

Comment:
Minor style question: I don't think you need to repeat "asset" in both the function name and the argument, especially when the type is also called Asset